2-mandatory/2-currency-code-lookup.js

```diff
@@ -17,10 +17,14 @@ const COUNTRY_CURRENCY_CODES = [
   ["MX", "MXN"],
 ];
 
+let newObject = {}
+
 function createLookup(countryCurrencyCodes) {
-  // write code here
+    let toObject = Object.fromEntries(countryCurrencyCodes);  
```

Comment:
How would you do this without using Object.fromEntries()? 
I think we went through this during the Saturday session.